api/src/main/java/org/pmiops/workbench/google/CloudStorageService.java

```diff
@@ -11,6 +11,7 @@
 
   public String readInvitationKey();
   public String readBlockscoreApiKey();
+  public String readMandrillApiKey();
```

Comment:
Not a nit with this code since it is following convention, but with our interfaces in general. Most methods on interfaces (abstract, default, and static) are public by default so they don't need the public modifier.